code/bit_manipulation/src/addition_using_bits/addition_using_bits.c

```diff
@@ -0,0 +1,49 @@
+/*
+Problem Statement : 
+Add two numbers using bitwise operators
+(i.e without using arithmetic operators)
+*/
+
+#include<stdio.h>
```

Comment:
```suggestion
#include <stdio.h>
```

Reply:
The two files addition_using_bits.c and addition_using_bits.java got added by mistake.
These files are already merged in other PR #4727.
Should I close this PR and raise a new one for file bit_division.cpp or this PR should be fine ??